widgets/BigQueryDelete-action.json

```diff
@@ -0,0 +1,78 @@
+{
+  "metadata": {
+    "spec-version": "1.5"
+  },
+  "display-name": "Google BigQuery Delete",
+  "configuration-groups": [
+    {
+      "label": "Basic",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Project ID",
+          "name": "project",
+          "widget-attributes": {
+            "default": "auto-detect"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Dataset Project ID",
+          "name": "datasetProject",
+          "widget-attributes": {
+            "placeholder": "Project the dataset or table belongs to, if different from the Project ID."
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Dataset Name",
+          "name": "dataset",
+          "widget-attributes": {
+            "placeholder": "Name of Dataset"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Table Name",
+          "name": "table",
+          "widget-attributes": {
+            "placeholder": "Name of Table"
+          }
+        },
+        {
+          "name": "deleteContents",
+          "label": "Delete Contents",
```

Comment:
'Delete Dataset Contents' would be more descriptive. Otherwise it can be confusing what this means when a table is specified.